docker/Ubuntu18.04-pytorch1.12.1-cuda11.3-cudnn8/Dockerfile

```diff
@@ -51,8 +51,9 @@ RUN wget -P /opt https://downloads.xiph.org/releases/flac/flac-1.3.2.tar.xz  &&
     find /opt/flac-1.3.2  -type f \( -name "*.o" -o -name "*.la" -o -name "*.a" \) -exec rm {} \; && \
     cd - 
 
-RUN pip install kaldiio graphviz && \
-	conda install -y -c pytorch torchaudio
+RUN conda install -y -c pytorch torchaudio=0.12 && \
+    pip install kaldiio graphviz
```

Comment:
Do we need kaldiio?

Reply:
To be honest, I am not sure where kaldiio is used. I included it there because the previous Dockerfile had kaldiio. 
Should I remove it?

Reply:
Yes, thanks!

Reply:
Okay. I've removed it in both Dockerfiles.